### PR TITLE
Add support of returning to challenge list

### LINKED ISSUE #2: I want return back to challenge list without app restart
Add 'b/backspace' button handling to return to list

### PR DESCRIPTION
**Pull request by mortum5**
Added a callback that allows the child model to transfer execution to the parent model
Resolves #2

### CODE REVIEW

File: play/play.go [hunk 1]
```diff
@@ -28,13 +28,17 @@ type model struct {
...
+	parentCallback Callback
+
 	help help.Model
 }
 
 const copyStatus = "Code has copied to clipboard"
 const copyErrStatus = "Code has not copied. Please install xsel, xclip, wl-clipboard or Termux:API"
 
-func New(c challenge.Challenge, width, height int) (tea.Model, tea.Cmd) {
+type Callback func(tea.Msg) (tea.Model, tea.Cmd)
```
Comment (rusinikita) [inline play/play.go]: 1. Let's rename it to BackHandler
2. Let's change argument type to tea.WindowSizeMsg

File: play/play.go [hunk 2]
```diff
@@ -28,13 +28,17 @@ type model struct {
 	question questionModel
 	learn    learn
 
+	parentCallback Callback
```
Comment (rusinikita) [inline play/play.go]: let's rename it to backHandler

**Review by rusinikita — CHANGES_REQUESTED** [review]
It works! Thank's a lot

Let's make few changes and it's ready to merge.

Important: Squash all your commits into one "Add back key"

File: play/play.go (hunk 1 above)
Comment (mortum5) [reply play/play.go]: Done

File: play/play.go (hunk 2 above)
Comment (mortum5) [reply play/play.go]: Done

**Comment by rusinikita** [comment]
🎉🎉🎉 Thanks! Please, contact me in TG to get reward

**Pull request merged**